packages/lodestar-cli/src/cmds/beacon/testnets/index.ts

```diff
@@ -0,0 +1,47 @@
+import fs from "fs";
+import path from "path";
+import stream from "stream";
+import {promisify} from "util";
+import got from "got";
+import {IBeaconNodeOptions} from "@chainsafe/lodestar/lib/node/options";
+import {IBeaconArgs} from "../options";
+
+type Testnet = "altona";
+
+export function getTestnetConfig(testnet: Testnet): Partial<IBeaconNodeOptions> {
+  switch (testnet) {
+    case "altona":
+      return JSON.parse(fs.readFileSync(path.join(__dirname, "altona.json"), "utf8"));
+    default:
+      throw Error(`Testnet not supported: ${testnet}`);
+  }
+}
+
+function getGenesisFileUrl(testnet: Testnet): string {
+  switch (testnet) {
+    case "altona":
+      return "https://github.com/eth2-clients/eth2-testnets/raw/master/shared/altona/genesis.ssz";
```

Comment:
use permalink to git hash, so we don't break if they rearrange paths

Reply:
That said, genesis file is always the same but we should probably fetch bootnodes (merge with some hardcoded in config) from repo since they tend to change as peoples nodes go down.

Reply:
Switched to permalink. Where can we fetched the updated bootnodes from?

Reply:
https://github.com/eth2-clients/eth2-testnets/blob/master/shared/altona/bootstrap_nodes.txt

Reply:
added